anta/tests/cvx.py

```diff
@@ -87,3 +96,137 @@ def test(self) -> None:
         cluster_status = command_output["clusterStatus"]
         if (cluster_state := cluster_status.get("enabled")) != self.inputs.enabled:
             self.result.is_failure(f"Management CVX status is not valid: {cluster_state}")
+
+
+class VerifyCVXClusterStatus(AntaTest):
+    """Verifies the CVX Server Cluster status.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the CVX Server Cluster is enabled.
+    * Failure: The test will fail if any of the following conditions are met:
+        - If the CVX Status is disabled
+        - If the peers are not in "Registration ok" state
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.cvx:
+      - VerifyCVXClusterStatus:
+          enabled: true
+          cluster_mode: true
+          role: Master
+          peer_status:
+            - peer_name : cvx-red-2
+              registration_state: Registration complete
```

Comment:
Sometimes, some switches can intentionally be left at a non-"registration complete" state in case the system is between upgrades. In these cases, I would like to provide an option to input that as part of test.

Reply:
you can keep the input but my point was that this input is the default so not needed.

Also according to the docstring you wrote I feel like if left on purpose in  non-"registration complete" would imply failure:

```
 * Failure: The test will fail if any of the following conditions are met:
        - If the CVX Status is disabled
        - If the peers are not in "Registration ok" state
``` 

Maybe it should be rewritten

```
        - If the peers are not in the expected state
```

Reply:
you may leave the registration_state in the docstring, my point is more to show some state other than the default one as this one can be omitted (because it is default) and so it is possible to show a test entry _without_ it.